src/main/kotlin/no/skatteetaten/aurora/herkimer/controller/ApplicationDeploymentController.kt

```diff
@@ -66,6 +84,34 @@ class ApplicationDeploymentController(
         }
     }
 
+    @PatchMapping("/{id}")
+    fun migrate(
+        @PathVariable id: PrincipalUID,
+        @Valid @RequestBody payload: ApplicationMigrationPayload
+    ): AuroraResponse<ApplicationDeployment> {
+        payload.validateAtLeastOneFieldIsSet()
+
+        val existingAd = principalService.findApplicationDeployment(id)
+            ?: throw NoSuchResourceException("Could not find ApplicationDeployment with id=$id")
+        return payload.run {
+            principalService.updateApplicationDeployment(
+                existingAd.copy(
+                    businessGroup = assignPropertyWithFallback(businessGroup, existingAd.businessGroup),
```

Comment:
jeg likte selv bedre `businessGroup = businessGroup ?: existingAd.businessGroup`

Hvis du ikke liker payload.run som assigner payload til `this` så kan du fjerne den. Da vil det kanskje bli litt klarere. For meg er det det samme.

Reply:
Jeg likte også det originale bedre, endrer tilbake